test/EntityFramework/FunctionalTests/Query/LinqToEntities/StringConcatTests.cs

```diff
@@ -194,7 +194,7 @@ public void Issue1904_StringConcatPlus_can_concat_constant_string_with_property_
                     Assert.Equal("a" + projection.Entity.BoolProp + "b", projection.BoolProp);
                     Assert.Equal("a" + projection.Entity.GuidProp + "b", projection.GuidProp);
                     Assert.Equal("a" + projection.Entity.FloatProp.ToString(NumberFormatInfo.InvariantInfo) + "b", projection.FloatProp);
-                    Assert.Equal("a" + projection.Entity.DoubleProp.ToString(NumberFormatInfo.InvariantInfo) + "b", projection.DoubleProp);
+                    Assert.Equal("a" + projection.Entity.DoubleProp.ToString("G15", NumberFormatInfo.InvariantInfo) + "b", projection.DoubleProp);
```

Comment:
I wonder if SQL Server will ever take the breaking change...

Reply:
No comment. ;-)